IPHONEX.md

```diff
@@ -0,0 +1,224 @@
+    This does not require any tooling, apart from a text editor.
+
+1. By going through the `Mobile App` flow in the Mendix Portal and selecting `Do it yourself` (advanced workflow)
+
+    An alternative approach is to make the changes in `config/resources.json`, and to follow the instructions in the `README.md` file to create a new `phonegap.zip`.
+    This approach is more complex, but allows for the most flexibility and customizations.
+
+### Configure the storyboard images using the simple workflow
+
+Place the new storyboard images (using the filenames outlined below) in the `res/android` or `res/ios` folder of your `phonegap.zip` file.
+
+Remove any `<splash ... />` configuration lines within the `<platform name="ios">` element of `config.xml`.
+Add new configuration lines according to the instructions below.
+
+Rezip everything, and upload the file to Phonegap Build.
+
+### Configure the storyboard images using the advanced workflow
+
+Place the new storyboard images (using the filenames outlined below) in the `src/resources/android` or `src/resources/ios` folder of your hybrid project folder (the folder in which the file you're currently reading resides).
+
+Remove any `{"tag": "splash",` configuration lines within the `"iosImages"` element of `config/resources.json`.
+Add new configuration lines according to the instructions below.
+
+Build a new `phonegap.zip` according to the instructions in the README file, i.e. by running `npm run package`.
+
+## Launch storyboard images
+
+In order to support newer form factors and split-screen/slide-over multitasking, you should use launch storyboard images. These are similar to the legacy launch images above, but there are crucial differences:
+
+* images are not specific to a given device.
+* images are scaled to fill the available viewport (while maintaining the aspect ratio).
+* the outer edges of the images will be cropped, and the amount will vary based on device an viewport.
+* there is no need to provide an image for each possible device, viewport, and orientation; iOS will choose the best image for the situation automatically.
+
+### Designing launch storyboard images
+
+The key to designing a launch storyboard image is understanding that the edges of the image will almost certainly be cropped.
+Therefore, one should not place any important information near the edges of any images provided to the launch storyboard.
+Only the center is a safe area, and this all but guarantees that following Apple's advice of presenting an unpopulated user interface will not work well.
+
+Instead, the following tips should enable you to create a launch image that works across a multitude of form factors, viewports, and orientations:
+
+* Important graphics (logos, icons, titles) should be centered. The safe bounding region will vary, so you will need to test to ensure that the important graphics are never cropped. Better yet, don't supply any important graphics in the first place.
```

Comment:
>> don't supply
supplying graphics to an image, is that idiomatic?

Reply:
Improved it